Makefile

```diff
@@ -1,15 +1,22 @@
+#get latest git tag
+TAG = $$(git describe --tags)
```

Comment:
Could you please change it to`git describe --abbrev=0 --tags`?

Reply:
fatto ma puoi scrivermi anche in italiano :)